includes/class-wc-payments-account.php

```diff
@@ -329,7 +329,7 @@ public function get_deposit_schedule_weekly_anchor(): string {
 	 */
 	public function get_deposit_schedule_monthly_anchor() {
 		$account = $this->get_cached_account_data();
-		return $account['deposits']['monthly_anchor'] ?? null;
+		return ! empty( $account['deposits']['monthly_anchor'] ) ? $account['deposits']['monthly_anchor'] : null;
```

Comment:
The `??` operator wasn't effective here because in some circumstances `$account['deposits']['monthly_anchor']` was an empty string, i.e. `''`.